lib/LLVMIRCodeGen/BundleSaver.cpp

```diff
@@ -52,6 +52,9 @@ static const char *headerFileTemplate =
 
 #include <stdint.h>
 
+// Glow bundle error codes
+#define GLOW_BUNDLE_SUCCESS 0
```

Comment:
IIRC, you planned to support more error-codes, eventually backend-specific error codes?

Reply:
Yes in the end I need error codes for our custom functionalities (DMA, RPCs).

Reply:
Do you plan on providing some mechanisms for defining them without changing the bundle-saver string constant, i.e. some way for a backend to define those and they would be just dumped here as defines?

Reply:
Yeah I want to do it and I was thinking how exactly ... the problem is about adding a text file into a buffer statically at compile time... Maybe use CMake to serialize a text file and then add it to a `char[]` buffer at compile time with `#include`? What do you think?

Reply:
What if you add an API like `virtual std::string getCustomBundleHeader()`, which each backend can override? By default it would return an empty string. But backends may return whatever they want. This content will be inserted somewhere in the emitted header of the bundle, e.g. after the standard part produced by the bundle saver now.

Reply:
Part of the LLVMIRGen class right?

Reply:
That is a great idea. I've done it. I actually named the function `getBundleHeaderExtra()`.

Reply:
> Part of the LLVMIRGen class right?

Yes, probably. Or LLVMBackend. But LLVMIRGen probably makes more sense.